src/editables/__init__.py

```diff
@@ -1,36 +1,12 @@
-from pathlib import Path
-TEMPLATE = """\
+import inspect
 import os
-import sys
-import importlib.util
-from importlib.machinery import FileFinder, SourceFileLoader, SOURCE_SUFFIXES
-
-name = __name__
-location_of_replacement = {location!r}
-excludes = {excludes!r}.split(",")
-
-class BackendImporter(FileFinder):
-    \"""Allow imports included, disallow import excluded\"""
-
-    def find_spec(self, fullname, target=None):
-        if any(fullname.startswith(m) for m in excludes.split(",")):
-            raise ImportError(fullname + " is excluded from packaging")
-        return super().find_spec(fullname, target)
+from pathlib import Path
 
-def finder(path):
-    if path.startswith(os.path.join(location, __name__)):
-        return BackendImporter(path, (SourceFileLoader, SOURCE_SUFFIXES))
-    raise ImportError
+from . import install_hook
```

Comment:
We shouldn't be importing the hook here. I guess this is an artifact of using `inspect.getsource`? Can it be done without importing?

Reply:
Can be a future improvement.